src/main/java/org/mule/service/http/impl/service/server/grizzly/ResponseStreamingCompletionHandler.java

```diff
@@ -83,7 +87,9 @@ private int calculateBufferSize(FilterChainContext ctx) {
     } else {
       LOGGER.debug("Transfer encoding header present, using fixed buffer size.");
     }
-    LOGGER.debug("Response streaming chunk calculated buffer size = {} bytes.", bufferSize);
+    if (LOGGER.isDebugEnabled()) {
```

Comment:
Is this necessary? There's only one parameter that is already calculated and since it's the logging formatting being used the full string shouldn't be created if debug is not enabled.

Reply:
Yep, apparently the number is converted to string before calling the logger:

![image](https://user-images.githubusercontent.com/8010105/47226941-12480680-d398-11e8-86fa-0a484a7fe12c.png)